docker/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:20.04
```

Comment:
Because it's the version you use with github actions https://github.com/daanx/mimalloc-bench/blob/master/.github/workflows/all.yml#L38 and this version seems to have minimal number of allocators/tests broken as I compare https://github.com/daanx/mimalloc-bench/blob/master/.github/workflows/all.yml#L54 to the counterparts